From ca405053a1a8d563ca6c4741246becb4f96ed150 Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Tue, 28 May 2024 13:14:44 +0800
Subject: [PATCH 037/223] fix(llc_spram):Removed linux version check

Changelogs:
1.Remove the KERNEL_VERSION check from llc_spram.c since it is
  a builtin driver.
---
 drivers/memory/eswin/codacache/llc_spram.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/memory/eswin/codacache/llc_spram.c b/drivers/memory/eswin/codacache/llc_spram.c
index 01744360937c..e84f3650ae37 100644
--- a/drivers/memory/eswin/codacache/llc_spram.c
+++ b/drivers/memory/eswin/codacache/llc_spram.c
@@ -103,11 +103,10 @@ struct spram_dev {
 	struct reset_control *rstc_llc;
 };
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,6,0)
 #define dma_buf_map		iosys_map
 #define dma_buf_map_set_vaddr	iosys_map_set_vaddr
 #define dma_buf_map_clear	iosys_map_clear
-#endif
+
 #define spram_phys_to_virt(spram, phys) \
 	 (spram->virt_base + phys - spram->phys_addr)
 
-- 
2.47.0

